PLBenchmarks/targets.py

```diff
@@ -0,0 +1,373 @@
+"""
+targets.py
+Functions and classes for handling the target data.
+"""
+
+import os
+import yaml
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import networkx as nx
+
+from . import __path__, ligands, edges, utils
+
+
+data_path = os.path.abspath(
+    os.path.join(os.path.join(__path__[0], "sample_data"))
+)
+file = open(os.path.join(data_path, "targets.yml"))
+target_dict = yaml.full_load(file)
+print(target_dict)
```

Comment:
This appears to be unnecessary (unless you want this dict of sample data printed when loading this module) and/or just for debugging

Reply:
Yes, already removed it.